test/Microsoft.Azure.WebJobs.Extensions.ServiceBus.Tests/Config/ServiceBusOptionsTests.cs

```diff
@@ -28,7 +28,7 @@ public ServiceBusOptionsTests()
         public void Constructor_SetsExpectedDefaults()
         {
             ServiceBusOptions config = new ServiceBusOptions();
-            Assert.Equal(16, config.MessageHandlerOptions.MaxConcurrentCalls);
+            Assert.True(config.MessageHandlerOptions.MaxConcurrentCalls >= 16);
```

Comment:
Recommend making this check stronger and compute the expected value in the test based on processor count. For the E2E test as well.

Reply:
Just fix these two test issues, and the code looks good to me